tests/orquestra/integrations/cirq/conversions/circuit_conversions_test.py

```diff
@@ -109,6 +116,8 @@ class TestGateConversion:
     def test_matrices_of_corresponding_orquestra_and_cirq_gates_are_equal(
         self, orquestra_gate, cirq_gate
     ):
+        if orquestra_gate == _wavefunction_operations.ResetOperation:
```

Comment:
What do you think about just making a separate test just for reset operation conversion? I think it might create confusion to include the reset operation in `EQUIVALENT_NON_PARAMETRIC_GATES` since it is not actually a gate.